core/dns.go

```diff
@@ -202,11 +205,11 @@ func (dnsResolver *DNSResolver) LookupCAA(domain string, alias bool) ([]*dns.CAA
 			caaR, ok := answer.(*dns.CAA)
 			if !ok {
 				err = errors.New("Badly formatted record")
-				return nil, err
+				return nil, totalRtt, err
```

Comment:
It seems like the definition of RTT is kind of confusing here. Why are you doing aggregate and not mean?